Author: Michael R. Crusoe <crusoe@debian.org>
Description: skip the furo docs theme; it is no packaged yet for Debian
Forwarded: not-needed
--- mypy.orig/docs/requirements-docs.txt
+++ mypy/docs/requirements-docs.txt
@@ -1,2 +1,2 @@
 sphinx>=4.2.0,<5.0.0
-furo>=2022.3.4
+sphinx-rtd-theme>=1.0.0,<2.0.0
--- mypy.orig/docs/source/conf.py
+++ mypy/docs/source/conf.py
@@ -103,7 +103,13 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "furo"
+try:
+    import sphinx_rtd_theme
+except:
+    html_theme = 'default'
+else:
+    html_theme = 'sphinx_rtd_theme'
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
--- mypy.orig/mypyc/doc/conf.py
+++ mypy/mypyc/doc/conf.py
@@ -50,7 +50,13 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'furo'
+try:
+    import sphinx_rtd_theme  # type: ignore
+except:
+    html_theme = 'default'
+else:
+    html_theme = 'sphinx_rtd_theme'
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
